src/Hpack/Config.hs

```diff
@@ -501,6 +509,21 @@ instance Semigroup Empty where
 instance FromValue Empty where
   fromValue _ = return Empty
 
+data GhcLanguage =
+    Haskell98
+  | Haskell2010
+  | GHC2021
+  deriving (Eq, Show, Generic, Enum, Bounded)
```

Comment:
What about using a `String` instead and leaving anything further to `Cabal`?   That way older versions of `hpack` will continue to work with the next set of `GHC202x` extensions.

A `newtype` will still be neat.  Could we call it `Language` instead of `GhcLanguage`, or is that already taken?

Reply:
I will change it to a `newtype` of a `String`, and name the type/constructor `Language`.